ui/src/components/DevPanel/DevPanel.tsx

```diff
@@ -1,17 +1,12 @@
 import React from 'react'
 import { DevToolsPanel } from '@tesler-ui/core'
-import { useSelector } from 'react-redux';
+import { useSelector } from 'react-redux'
 import { AppState } from '../../interfaces/storeSlices'
 
 const DevPanel: React.FunctionComponent = () => {
     const showCondition = useSelector((state: AppState) => state.session.devPanelEnabled)
-    return (
-        <>
-            {showCondition && (
-                <DevToolsPanel/>
-            )}
-        </>
-    )
+    if (!showCondition) return null
```

Comment:
Missing brackets; why there is no linter rules for brackets?

Reply:
@Dergash "curly" rule is added